Automattic/wp-calypso
# Disable OAuth login on desktop and enable native WpLogin instead

This patch does what the title says it does. It also updates the way we logout the user, calling an async logout endpoint (`/wp-login.php?action=logout-endpoint`) instead of redirecting the user to logout_URL.

Review [`wp-desktop` #318](https://github.com/Automattic/wp-desktop/pull/318) first 

Then check that there is no regression on logout with this change:

### Testing instructions
- Apply patch D6403 on the server
- Checkout and boot this branch locally `git checkout update/desktop-login && npm start`
- Check that going to http://calypso.localhost:3000/oauth-login now redirects to `/`
- Go to http://calypso.localhost:3000/me and click on `Sign out`
- Check that it still logs you out properly and that you are redirected to `/` (which should redirect to `/devdocs/start` on your local dev env)

### Reviews
